fe1-web/components/__tests__/TextInputChirp.test.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react-native';
+import STRINGS from 'res/strings';
+import TextInputChirp from '../TextInputChirp';
+
+const onChangeText = jest.fn();
+const onPress = jest.fn();
```

Comment:
you might want to have these initialized in a `beforeEach` call, to avoid mocks being shared across tests (and thus making the tests not-independent)

Reply:
True, I didn't think about that since my functions don't do anything.